test/specs/utils/cryptoModule.js

```diff
@@ -174,6 +174,38 @@ describe('Crypto class', () => {
 							});
 						});
 					});
+					describe('#verifyMessage', () => {
+						Given(
```

Comment:
Prefer using the same message/signature/public key, but in any case it makes no sense to nest this suite under `Given a message "Hello Lisker"` if we're not using it.